packages/api/package.json

```diff
@@ -65,7 +65,7 @@
     "lint:fix": "yarn run lint --fix",
     "pretest": "yarn run check-types",
     "test": "yarn test:unit && yarn test:e2e",
-    "test:unit": "vitest --run --dir test/unit/ --coverage",
```

Comment:
I know you were pushing for coverage recently.  I noticed this was turned off everywhere.  What was the decision to do this driven by?

Reply:
It's just moved to the config file, so so we can control it via `process.env.CI` variable. This helps to run tests faster locally for quick developer feedback.